### PR TITLE
Incorporate mongo feedback

### PR DESCRIPTION
**Pull request by umbauk**
- Incorporate Shums feedback from previous mongoDB PR including add a config.js file with MongoDB URI info. We'll likely use this to store api keys in future and should add to .gitignore when we do.

- Add a populateUsers.js script which populates db.users collection with 3 dummy users for development.

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -4,12 +4,13 @@ const { join } = require("path");
...
 const logger = require("morgan");
 const mongoose = require("mongoose");
+const config = require("./config.js");
 
 const indexRouter = require("./routes/index");
 const pingRouter = require("./routes/ping");
 
 // Connect to the database
-const mongoDB = "mongodb://127.0.0.1/mail-sender-dev";
+const mongoDB = config.mongoURI + ":" + config.mongoPort + "/" + config.mongoDB;
```
Comment (skassam21) [inline server/app.js]: nitpick: generally recommend template strings (``) vs. concatenation like this

File: server/config.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+let config = {};
```
Comment (skassam21) [inline server/config.js]: should use const here - generally favour const over let, especially if you don't want to rebind the value 

Also, may be cleaner to write it like this:
```
const config = {
    mongoURI: process.env.MONGO_URI || "mongodb://localhost", 
    mongoPort: process.env.MONGO_PORT || "27017",
    mongoDB = process.env.MONGO_DB || "mail-sender-dev"
}
```

**Review by skassam21 — APPROVED** [review]

File: server/config.js (hunk 2 above)
Comment (umbauk) [reply server/config.js]: Ofcourse! I should have thought before copying it verbatim from stackoverflow :)
Fixed.

File: server/app.js (hunk 1 above)
Comment (umbauk) [reply server/app.js]: Makes sense. Fixed.

**Pull request merged**